src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+body,
+.nav {
+  margin: 0;
+  padding: 0;
+  max-width: 1200px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.nav {
+  justify-content: space-between;
+  display: flex;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.nav_link {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider setting the font family for the entire page to get it inherited by elements inside